src/Umbraco.Core/Configuration/Models/GlobalSettings.cs

```diff
@@ -82,8 +82,8 @@ internal const string
     public string UmbracoPath
     {
         get => Constants.System.DefaultUmbracoPath;
-        [Obsolete($"{nameof(UmbracoPath)}  is no longer configurable, property setter is scheduled for removal in V12")]
-        // NOTE: when removing this, also clean up the hardcoded removal of UmbracoPath in UmbracoJsonSchemaGenerator
+        [Obsolete($"{nameof(UmbracoPath)} is no longer configurable, this property setter is scheduled for removal in V12.")]
+        // NOTE: When removing this, also clean up the hardcoded removal of UmbracoPath in Umbraco.JsonSchema
         set { }
     }
```

Comment:
Why is only the setter marked as obsolete? There's no reason for only keeping the getter in future versions and obsoleting the whole property would ensure we update all references to the constant value, possibly removing the need to inject the `IOptions[Monitor|Snapshot]<GlobalSettings>` into implementations that only require this value.

Obsoleted properties are also ignored when generating the JSON schema, so that would already allow cleaning up the hardcoded removal of this property.